addons/hr_timesheet/static/src/components/task_with_hours/task_with_hours.js

```diff
@@ -43,7 +43,7 @@ class TaskWithHours extends Many2OneField {
         super.computeActiveActions(props);
         const activeActions = this.state.activeActions;
         activeActions.create = activeActions.create && this.canCreate;
-        activeActions.createEdit = activeActions.create;
+        activeActions.createEdit = activeActions.create && props.canCreateEdit;
```

Comment:
Why not doing that: 

```suggestion
        activeActions.createEdit = activeActions.createEdit && this.canCreate;
```
? 🙂

Reply:
Done!!